bigframes/__init__.py

```diff
@@ -22,6 +24,20 @@
 from bigframes.session import connect, Session
 from bigframes.version import __version__
 
+
+def load_ipython_extension(ipython):
```

Comment:
Consider adding an unload function that undoes any bigframes-specific configuration of `%%bigquery`

See: `unload_ipython_extension` in https://ipython.readthedocs.io/en/stable/config/extensions/#writing-extensions